src/quick-start.md

```diff
@@ -81,29 +86,26 @@ The complete documentation of this Custom Resource can be found
 or on
 [docs.crds.dev](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller).
 
-> **NOTE:** Admission policies are registered with a `*` webhook
+> **NOTE:** ClusterAdmissionPolicy resources are registered with a `*` webhook
 > `scope`, which means that registered webhooks will be forwarded all
 > requests matching the given `resources` and `operations` -- either
 > namespaced (in any namespace), or cluster-wide resources.
 
-> **NOTE:** The `AdmissionPolicy` resource is cluster-wide. There are
+> **NOTE:** The `ClusterAdmissionPolicy` resource is cluster-wide. There are
 > plans to also provide a namespaced version that will only impact
 > registered namespaced resources on its own namespace.
 
 ## Enforce your first policy
 
 We will use the [`pod-privileged` policy](https://github.com/kubewarden/pod-privileged-policy).
```

Comment:
The name (line 45) is `privileged-pods` ?!

Reply:
Good catch, I've fixed it now